docs/CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ The following `tip` changes can be tested by building VictoriaMetrics components
 * [How to build vmauth](https://docs.victoriametrics.com/vmauth.html#how-to-build-from-sources)
 * [How to build vmctl](https://docs.victoriametrics.com/vmctl.html#how-to-build)
 
+Metrics of the actual version of VictoriaMetrics cluster are available for viewing at our 
```

Comment:
Is `actual` in this context referring to `latest`?
It seems to me like it will be using `latest` here instead, if that's the idea.